JSDemos/Demos/Charts/Strips/description.md

```diff
@@ -0,0 +1,5 @@
+A strip is a colored piece of the **Chart**'s background that highlights a range of values.
+
+To configure strips, declare an array of objects in the **argumentAxis**.[strips[]](Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/strips/) or **valueAxis**.[strips[]](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/) option. Each object in this array configures and individual strip. This demo shows two strips on the value axis, and how to configure the [label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/)'s [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/#text) and **font**.[color](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/font/#color) options for each strip separately.
+
+You can specify the appearance of the strips in the [stripStyle](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/) object for the value axis, argument axis, or for both axes in the [commonAxisSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/) object. Individual settings override common settings. In this demo, we specify the font [weight](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/label/font/#weight) and [size](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/label/font/#size) options for all strips labels on the value axis. 
```

Comment:
Fixing the following issues: 

- stripStyle is for specifying the same appearance for multiple strips.
- The first sentence had too much information.
- The "in this demo" part wasn't connected to the rest of the paragraph.

```suggestion
You can specify the same appearance for all strips on a specific axis or in the **Chart**. To do this, declare the [stripStyle](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/) object in the [valueAxis](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/), [argumentAxis](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/), or [commonAxisSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/) object. Individual settings override common settings. In this demo, we use the **stripStyle** object to specify the font [weight](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/label/font/#weight) and [size](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/label/font/#size) for all strip labels on the value axis. 
```